modules/checklist/module.php

```diff
@@ -143,6 +146,14 @@ public function enqueue_editor_scripts() : void {
 		} );
 	}
 
+	public static function is_preference_switch_on() : bool {
+		$user_preferences = get_user_meta( get_current_user_id(), Preferences_Manager::META_KEY, true );
```

Comment:
Try to use `$editor_preferences->get_model()->get_settings( self::VISIBILITY_SWITCH_ID )`

Reply:
Done